lib/Mojo/Server/Morbo.pm

```diff
@@ -46,12 +37,15 @@ sub run {
   exit 0;
 }
 
-sub _list { path(shift)->list_tree->map('to_string')->each }
+sub watch {
+  deprecated 'watch is deprecated in favor of backend->watch';
+  shift->backend->watch(@_);
+}
 
 sub _manage {
   my $self = shift;
 
-  if (my @files = @{$self->modified_files}) {
+  if (my @files = @{$self->backend->modified_files}) {
```

Comment:
_manage() has `sleep 1;` in the bottom. Could it be a backend task to sleep? Was thinking it might be faster with the modules that has event based file changes.